go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	code.cloudfoundry.org/go-loggregator/v8 v8.0.5
 	code.cloudfoundry.org/gofileutils v0.0.0-20170111115228-4d0c80011a0f
 	code.cloudfoundry.org/jsonry v1.1.4
-	code.cloudfoundry.org/lager v1.1.1-0.20191008172124-a9afc05ee5be
+	code.cloudfoundry.org/lager v2.0.0+incompatible
```

Comment:
hm, what this `+incompatible` mean?

Reply:
@moleske ?

Reply:
https://stackoverflow.com/questions/57355929/what-does-incompatible-in-go-mod-mean-will-it-cause-harm

means that it doesn't use go.mod correctly or go expected semvar correctly

Reply:
we already bumped to lager v2 in v7
https://github.com/cloudfoundry/cli/commit/e12c8456e168fc798516063fd2326b7b025ba7b3